### DISCUSSION TITLE
Tag prop inside variants doesn't change the underlying HTML element

### DISCUSSION THREAD

**Discussion by mimameid** [tamagui/tamagui#2037] [General]
I'd expect it to work, since a `tag` prop inline changes the HTML element. Tamagui Version: 1.76.0

What happened to the issue template btw.?

**Comment by natew** [comment]
Going to move to feature request, right now variants technically only do styles. But we do actually need to fix the types to be more strict.

**Comment by jpaas** [comment]
Any update on this one?
